theme/ckeditor5-ui/components/dropdown/dropdown.css

```diff
@@ -12,21 +12,23 @@
 	font-size: inherit;
 
 	/* A triangle displayed to indicate this is actually a dropdown. */
```

Comment:
We can think about removing comment because `.ck-dropdown__arrow` speaks for itself?